analyses/oncoprint-landscape/util/oncoplot-palette.R

```diff
@@ -29,4 +29,42 @@ color_palette <- c("Missense_Mutation" = "#35978f",
                    "loss" = "#0072B2",
                    "gain" = "#D55E00",
                    "High_Level_Gain" = "#FF0000",
-                   "Multi_Hit_Fusion" = "#CD96CD")
+                   "Multi_Hit_Fusion" = "#CD96CD",
+                   "Adenoma" = "#f23d3d",
+                   "ATRT" =	"#731d1d",
+                   "Central neurocytoma" = "#b38686",
+                   "Chondrosarcoma" =	"#cc5c33",
+                   "Chordoma" =	"#331c0d",
+                   "Choroid plexus tumor" =	"#ffb380",
+                   "CNS EFT-CIC" = "#b25f00",
+                   "CNS lymphoma"	= "#f2d6b6",
+                   "CNS neuroblastoma" = "#736556",
+                   "CNS Rhabdomyosarcoma"	= "#ffaa00",
+                   "CNS sarcoma" =	"#4c3d00",
+                   "Craniopharyngioma" =	"#e2f200",
+                   "DNET" =	"#919926",
+                   "Dysplasia" = "#d6f2b6",
+                   "Embryonal Tumor" = "#304d26",
+                   "Ependymoma" = "#00f241",
+                   "ETMR" =	"#009929",
+                   "Ganglioglioma" = "#698c7c",
+                   "Germinoma" = "#39e6c3",
+                   "Glial-neuronal tumor NOS" = "#005359",
+                   "Gliosis" =	"#263233",
+                   "Hemangioblastoma" =	"#00c2f2",
+                   "Hemangioma"	= "#40a6ff",
+                   "HGAT" =	"#406280",
+                   "Langerhans Cell histiocytosis" = "#0044ff",
+                   "LGAT" =	"#00144d",
+                   "LGMT" = "#acbbe6",
+                   "Medulloblastoma" = "#7373e6",
+                   "Meningioma" =	"#3d0099",
+                   "MPNST" = "#c200f2",
+                   "Neurofibroma" =	"#917399",
+                   "none" =	"#f1f1f1",
+                   "Oligodendroglioma" = "#f279da",
+                   "Other" = "#cc0052",
+                   "Pineoblastoma" = "#994d6b",
+                   "Schwannoma" =	"#4d2636",
+                   "Teratoma" =	"#ffbfd9"
+)
```

Comment:
If the palette in `figures/palettes/histology_color_palette.tsv` were to be updated, you would have to also make changes to this code. Instead, I would find a way to use `figures/palettes/histology_color_palette.tsv` and I'd even consider moving the OncoPrint palette to a TSV files in `figures/palettes`.

Reply:
Looks like this has come up before! 😅 https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/510#issuecomment-607523137 Probably means we should do it!